### PR TITLE
Update create-graphqls-file.md

### PR DESCRIPTION
**Pull request by slkra**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will update https://devdocs.magento.com/guides/v2.3/graphql/develop/create-graphqls-file.html#specify-sorting-attributes page with a wider description of SortEnum and a link to a reference.

## Additional information

List all affected URLs 

https://devdocs.magento.com/guides/v2.3/graphql/develop/create-graphqls-file.html#specify-sorting-attributes

### CODE REVIEW

File: guides/v2.3/graphql/develop/create-graphqls-file.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ input ProductSortInput {
 
 `ProductSortInput` indicates that the attributes are available to catalog (Product) queries. If you specify a module-specific value such as `VolumizerSortInput`, then the attribute will be available only to queries processed by that module.
 
-`SortEnum` is defined in the base `schema.graphqls` file.
+`SortEnum` defines the order of results returned (ASC or DESC). For the definition see [`schema.graphqls`](https://github.com/magento/magento2/blob/2.3/app/code/Magento/GraphQl/etc/schema.graphqls) file.
```
Comment (erikmarr) [inline guides/v2.3/graphql/develop/create-graphqls-file.md]: change the first part of the description to: "indicates whether to return results in ascending or descending order."

Comment (slkra) [reply guides/v2.3/graphql/develop/create-graphqls-file.md]: @erikmarr done!

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @slkra!

**Pull request merged**